delta/utils/solver/asr_solver.py

```diff
@@ -181,16 +184,25 @@ def model_fn(self, mode):
       # data must be (features, labels), only using features as input
       self.model.build(input_shape=self.batch_input_shape[0])
 
-    # parallel and compile model
-    self.build(multi_gpu=(mode == utils.TRAIN))
-
-    if mode != utils.TRAIN:
-      model_path = Path(self._model_path).joinpath('best_model.ckpt')
+    assert self._model_load_type in ['best', 'lastest', 'no']
```

Comment:
what is the meaning of `no`?

Reply:
`lastest` -> `latest`

Reply:
don't load existing models

Reply:
Does "no" mean the training from scratch? If so, it is better to use `scratch`.